src/explore-education-statistics-frontend/src/modules/data-catalogue/components/DataSetFileApiChangelog.tsx

```diff
@@ -6,10 +6,15 @@ import React from 'react';
 
 interface Props {
   changes: ApiDataSetVersionChanges;
+  publicGuidanceNotes?: string;
   version: string;
 }
 
-export default function DataSetFileApiChangelog({ changes, version }: Props) {
+export default function DataSetFileApiChangelog({
+  changes,
+  publicGuidanceNotes,
```

Comment:
Really minor, but would personally simplify this to just `guidanceNotes` as the 'public' part is fairly redundant. Can leave it though as well.